xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -54,7 +54,7 @@ To be able to set max depth while reading OR allow skipping comments, create an
       </Parameters>
       <Docs>
         <param name="jsonData">The UTF-8 encoded JSON text to process.</param>
-        <param name="options">An object that defines customized behavior of the <see cref="T:System.Text.Json.Utf8JsonReader" /> that differs from the JSON RFC (for example how to handle comments or maximum depth allowed when reading). By default, the <see cref="T:System.Text.Json.Utf8JsonReader" /> follows the JSON RFC strictly; comments within the JSON are invalid, and the maximum depth is 64.</param>
+        <param name="options">Defines customized behavior of the <see cref="T:System.Text.Json.Utf8JsonReader" /> that differs from the JSON RFC (for example how to handle comments or maximum depth allowed when reading). By default, the <see cref="T:System.Text.Json.Utf8JsonReader" /> follows the JSON RFC strictly; comments within the JSON are invalid, and the maximum depth is 64.</param>
```

Comment:
`JsonReaderOptions` is not a class, but rather a struct. Hence, removed the use of "object".

Reply:
Not really necessary but the result sounds ok to me.

If you look at the MS documentation for "C# Objects", struct instances can also be called objects:

> _[...] Because structs are value types, a variable of a struct object holds a copy of the entire object. Instances of structs can also be created by using the new operator, but this is not required [...]_

Source: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/objects

Reply:
I still prefer the old version here.